frame/collective/src/lib.rs

```diff
@@ -708,10 +730,128 @@ decl_module! {
 			let proposal_count = Self::do_disapprove_proposal(proposal_hash);
 			Ok(Some(T::WeightInfo::disapprove_proposal(proposal_count)).into())
 		}
+
+		/// Re-dispatch a collective origin call as a unique account that represents that origin.
+		///
+		/// Since the collective may want to be able to represent a different ratio than exists,
+		/// we allow the re-dispatch to specify the target ratio, as long as it is less restrictive
+		/// than the provided origin.
+		///
+		/// Can only be called by the collective origin.
+		#[weight = {
+			T::WeightInfo::dispatch_as_ratio_account(*length_bound)
+			.saturating_add(proposal.get_dispatch_info().weight)
+		}]
+		fn dispatch_as_ratio_account(origin,
+			proposal: Box<T::Proposal>,
+			#[compact] length_bound: u32,
+			#[compact] target_n: u32,
+			#[compact] target_d: u32,
+		) -> DispatchResult {
```

Comment:
so this will live in the relay chain right? who is going to call this now?

Reply:
I'm probably wrong, this will live in the parachain.

Reply:
And the sequence of events is: 
1. A call `C` exists on the parachain that has a `EnsureSignedBy<X>` protecting its origin.
2. Polkadot governance would send `dispatch_as_ratio_account(Box<C>, _, 4, 5)` up to the parachain via XCM.
3. Once the XCM message is dispatched, it will try and execute `C` with the 4/5 origin that is generated. 

So we'd probably want something like `EnsureSignedByEitherOf` in a parachain? or do we already allow this?

Reply:
This will live on the relay chain.

The relay chain will use its governance to redispatch an XCM as this special account.

The parachain will have hardcoded this account as a powerful account with rights to control its governance.

Reply:
But that means we need to include the Collective pallet in the parachain, right?

Reply:
This is a different way compared to Gav's XCM Plurality origin, right?